scrapi/base/__init__.py

```diff
@@ -241,14 +250,10 @@ def normalize(self, raw_doc):
                 'ns0:header/ns0:setSpec/node()',
                 namespaces=self.NAMESPACES
             )
-            approved = False
-            for item in set_spec:
-                item_mod = item.replace('publication:', '')
-                if item_mod in self.approved_sets:
-                    approved = True
-                else:
-                    logger.info('Series {} not in approved list'.format(item))
-            if not approved:
+            # check if there's an intersection between the approved sets and the
+            # setSpec list provided in the record. If there isn't, don't normalize.
+            if not set([x.replace('publication:', '') for x in set_spec]).intersection(self.approved_sets):
```

Comment:
This can just be a set comprehension as mentioned by @fabianvf 
`{x.replace('publication:', '') for x in set_spec}`